pkg/handler3/handler.go

```diff
@@ -157,6 +193,27 @@ func (o *OpenAPIService) HandleDiscovery(w http.ResponseWriter, r *http.Request)
 	http.ServeContent(w, r, "/openapi/v3", time.Now(), bytes.NewReader(data))
 }
 
+func CacheBustingShouldRedirect(r *http.Request, etag string) bool {
```

Comment:
This function is only used once. I'm not sure keeping it separate it contributes much to code hygiene/readability

Reply:
Could be written as one-line:

```go
if hash := r.URL.Query().Get("hash); hash != "" && hash != etag {
    u := constructURL(group, etag)
	http.Redirect(w, r, u, 301)
	return
}
```

Reply:
Fixed